src/main/java/com/diffmin/patch/PatchApplication.java

```diff
@@ -90,6 +92,18 @@ private static void performInsertion(
                 types.add(where, (CtType<?>) toBeInserted);
                 inWhichCompilationUnit.setDeclaredTypes(types);
                 break;
+            case MODIFIER:
+                if (toBeInserted instanceof CtVirtualElement) {
+                    Set<ModifierKind> newModifiers = new HashSet<>();
+                    for (Object modifierKind : ((CtVirtualElement) toBeInserted).getChildren()) {
+                        newModifiers.add((ModifierKind) modifierKind);
+                    }
+                    ((CtTypeMember) inWhichElement).setModifiers(newModifiers);
```

Comment:
`addAll` was throwing typecasting errors as it was expecting `capture of <? extends ?>` but `capture of <?>` was given.